### ISSUE TITLE
getEvents retrieves only up to 10 events

### ISSUE THREAD

**Issue by boringusername99** [O365/python-o365#93]
Thanks for writing this library. It's simple to use.

When calling calendar.getEvents(), the API only returns a max of ten results. The events aren't even the ones that are coming up next - they are scattered between ones today, ones in the future, and ones in the past.

It seems like filtering and/or paging needs to implemented to retrieve more. It's a bit difficult to tell, since it looks like this library is also using v1.0 of the API, which is deprecated, and will be disabled in Nov 2018. The Microsoft docs are all for v2.0 and don't explain the functionality of 1.0.

**Comment by Narcolapser** [comment]
All true statements. We are currently working on a version 2 of this library that will use only v2.0 of the API, it will handle what you are trying to do much better. If you want to try the in development library you can check it out here:

https://github.com/O365/python-o365/tree/rewrite

**Comment by boringusername99** [comment]
I'm testing out your rewrite branch now, but having trouble importing. I installed with:
pip install git+https://github.com/O365/python-o365@rewrite

And I get:
```
~/.virtualenvs/rpi-office-notifier/lib/python3.6/site-packages/O365/account.py in <module>()
----> 1 from O365.connection import Connection, Protocol, MSGraphProtocol, BasicAuthProtocol, AUTH_METHOD
      2 from O365.utils import ME_RESOURCE
      3 from O365.message import Message
      4 from O365.mailbox import MailBox
      5 from O365.address_book import AddressBook, GlobalAddressList

~/.virtualenvs/rpi-office-notifier/lib/python3.6/site-packages/O365/connection.py in <module>()
     14 from requests_oauthlib import OAuth2Session
     15
---> 16 from O365.utils import ME_RESOURCE
     17
     18 log = logging.getLogger(__name__)

ModuleNotFoundError: No module named 'O365.utils'
```

If I download a zip instead, I can import O365 without any problems, although I prefer to install by pip for better organization of files within my project.

**Comment by Narcolapser** [comment]
Noted. Hopefully we'll have it properly install able via pip soon.

**Comment by hellbe** [comment]
@Narcolapser any indications on timeline for releasing the rewrite? New interface and functionality looks great!

**Comment by Narcolapser** [comment]
@hellbe So it Janscas had been working on it, but he seems to have disappeared. Hopefully I'll get around to using what he's done so far and expanding on it soon. The most important thing he had done was make this library OAuth compliant. But I'm glad to hear that other people like the new interface. It encourages me that I wasn't crazy when I was coming up with it. xD

@boringusername99 did Hellbe's comment help?

**Comment by janscas** [comment]
This is available in 1.0
